src/AppFooter.js

```diff
@@ -0,0 +1,14 @@
+function AppFooter({ changeButton, buttonClass, addToStorage}) {
```

Comment:
I don't think that your AppFooter should be aware of `addToStorage`, it should take 1 prop - onButtonChange

Reply:
there may be a bigger refactoring required with this component, however let's keep it for the future